logprep/abc/getter.py

```diff
@@ -19,7 +21,12 @@ class Getter(ABC):
 
     def get(self) -> str:
         """calls the get_raw method and returns the decoded content"""
-        return self.get_raw().decode("utf8")
+        content = self.get_raw().decode("utf8")
+        try:
+            content = Template(content).substitute(**os.environ)
+        except (KeyError, ValueError):
+            pass
+        return content
```

Comment:
It's not really a necessary change, but I think this is a nice case where we could make use of the `finally` clause:
```suggestion
        try:
            content = Template(content).substitute(**os.environ)
        finally:
            return content
```

Reply:
this could swallow exceptions, but I only want to handle explicitly these exceptions. All other exceptions should raise without returning content